plugins/mcreator-localization/lang/texts.properties

```diff
@@ -167,6 +167,7 @@ dialog.gui.button_add_title=Button editor
 dialog.gui.button_change_width=After you click OK, right-click with the mouse to change button width
 dialog.gui.button_resize=To resize the button, use move tool and right-click with the mouse when moving
 dialog.gui.button_text=Text of the button: 
+dialog.gui.is_text_button=Is a plain text button?
```

Comment:
I think users might not understand such wording. Maybe use some other words like _"undecorated"_, _"invisible"_ and/or _"hyperlink"_? For instance:
```properties
dialog.gui.is_text_button=<html>Is button undecorated?<br><small>Check to make the button base invisible

Reply:
This is not really invisible. This is another type of button where this is only a text you can click on it (a clickable label if you prefer)

Reply:
Some call such buttons "undecorated", I think Java Swing too, so this may be a word to use